reporter-web-app/src/components/PackageDetails.js

```diff
@@ -40,6 +40,18 @@ const PackageDetails = (props) => {
         vcsProcessed
     } = webAppPackage;
 
+    const {
```

Comment:
Commit message nits: s/reporter-webapp/reporter-web-app/

Can you also briefly explain how it was before, if not for each package?

Reply:
I don't see this being addressed in the commit message.